qwp_transport.go

```
@@ -0,0 +1,419 @@
+		// TLS configuration for wss:// connections.
+		dialOpts.HTTPClient = &http.Client{
+			Transport: &http.Transport{
+				TLSClientConfig: &tls.Config{
+					InsecureSkipVerify: true,
+					MinVersion:         tls.VersionTLS12,
+				},
+			},
+		}
+	}
+
+	conn, resp, err := websocket.Dial(ctx, wsURL, dialOpts)
```

golangci-lint (2.11.4)

[error] response body must be closed

(bodyclose)